### DISCUSSION TITLE
Why can't I download videos it keeps saying recovering

### DISCUSSION THREAD

**Discussion by Fiqa511** [TeamNewPipe/NewPipe#11283] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Why can't I download videos it keeps saying recovering

**Comment by wb9688** [comment, marked as answer]
Discussions are not the place to report bugs and this description is not enough information, which is why we have an issue template to help you tell us all the necessary details. You can report a bug through https://github.com/TeamNewPipe/NewPipe/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml, though please also first read https://github.com/TeamNewPipe/NewPipe/discussions/11260 about the current situation, which might already answer the questions you have.
